tests/oauth/test_base.py

```diff
@@ -61,7 +61,20 @@ def json(self):
 
 
 @pytest.mark.asyncio
-async def test_oauth(mocker):
+async def test_oauth_without_fastapi():
+    with pytest.raises(FastAPIImportError):
+        import spylib.oauth  # noqa
```

Comment:
Better to ignore specific error and not all of them. I think that's the right one:
```suggestion
        import spylib.oauth  # noqa: F401
```